core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -120,11 +120,48 @@ var SkillEditorPage = function() {
     });
   };
 
-  this.expectRubricExplanationToMatch = function(index, explanation) {
-    rubricExplanations.then(function(explanations) {
-      explanations[index].getText().then(function(text) {
-        expect(text).toMatch(explanation);
-      });
+  this.deleteRubricExplanationWithIndex = function(diffIndex, explIndex) {
+    var editRubricExplanationButtons = element.all(
+      by.css('.protractor-test-edit-rubric-explanation-' + diffIndex));
+    editRubricExplanationButtons.then(function(buttons) {
+      buttons[explIndex].click();
```

Comment:
Can we make sure these buttons have loaded before clicking?

Reply:
Done

Reply:
@aks681 If none of the buttons have loaded, will `buttons[explIndex]` work? For example, if `explIndex = 1`, then will evaluating `buttons[1]` give an index out of bounds error since no buttons will be found? I'm not sure how Protractor handles this.

Reply:
According to issues in the protractor repo [this](https://github.com/angular/protractor/issues/2314) and [this](https://github.com/angular/protractor/issues/2917), it seems it throws an index out of bounds error. 
They seem to workaround it by adding a custom browser.wait function. Should I try that here?